hack/test-e2e.sh

```diff
@@ -5,7 +5,7 @@ if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
   source "$(dirname $0)/common"
 fi
 
-IMAGE_CLUSTER_LOGGING_OPERATOR=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest}
+IMAGE_CLUSTER_LOGGING_OPERATOR=${IMAGE_CLUSTER_LOGGING_OPERATOR:-openshift/origin-cluster-logging-operator:latest}
```

Comment:
why?

Reply:
I'm seeing that the logcollector SA doesn't have the proper permissions even though I don't see this issue in my env. I'm testing to see if the issue is that we aren't actually testing the image being built. The correct fix would likely be in the release repo if that is the case.